README.md

```diff
@@ -41,13 +41,14 @@ Install as usual, see [this](https://www.drupal.org/docs/7/extend/installing-mod
 
 ## Usage
 
-Out of the box, Islandora usage stats provides:
-* views of usage stats on Collection overview pages
-* A report-generating interface at __Reports > Islandora Usage Stats Reports__
+The data collected by Islandora Usage Stats is made available to Views, so custom reports can be created. It also works with community modules such as [Islandora Usage Stats Callbacks](https://github.com/Islandora-Labs/islandora_usage_stats_callbacks) to provide more functionality. 
+
```

Comment:
Shameless self-promotion here, but if we include Callbacks, should we include https://github.com/mjordan/islandora_usage_stats_charts ?

Reply:
I'm down with that suggestion! Though you should contribute that to Labs if you think it's ready, @mjordan

Reply:
:+1: to including a like to @mjordan's module, and :+1: to him adding it to labs :smile:

Reply:
Bring them on, folks! (yes, and to any more that you have and are ready to publish! Sorry Mark, I had meant to add Charts in.